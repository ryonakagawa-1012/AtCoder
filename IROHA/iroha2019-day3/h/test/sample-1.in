7
a 48
t 27
c 30
o 1
d 58
e 88
r 800
