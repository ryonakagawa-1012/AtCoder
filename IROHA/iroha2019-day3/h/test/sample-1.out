aaaabcd
-1
abcdefghijklmnop
a
-1
abcdeeefghijkl
abcdefghijklmnopqrrrrrstuvwxyz
